script.js

```diff
@@ -18,40 +18,40 @@ const authorNames = () => {
     let authors = new Proxy({}, validator);
 }
 
-const api = (source) => fetch(`${BASE_URL}`+source+`&apiKey=${API_KEY}`); //Fetch api function
+const api = (source) => fetch(`${BASE_URL}${source}&apiKey=${API_KEY}`); //Fetch api function
+//Render News Function for creation of DOM
+const renderNewsItem = (news) => {    
```

Comment:
And I think will be better to have it as a method of NewsChannel class

Reply:
Agreed and Applied.